src/main/java/org/jclouds/vcloud/director/v1_5/domain/RasdItemsList.java

```diff
@@ -42,163 +47,163 @@
  */
 @XmlRootElement(name = "RasdItemsList")
 @XmlType(name = "RasdItemsList")
-public class RasdItemsList extends Resource implements Set<RasdItem> {
+public class RasdItemsList extends ForwardingList<RasdItem> implements Set<RasdItem> {
```

Comment:
It's not really imlementing the `Set` contract, just delegating to the items list.